benchmarks/bench_parsing.py

```
@@ -0,0 +1,194 @@
+    total_chars = 0
+    total_files = 0
+    errors = 0
+    timings = []
+
+    for fpath in files:
+        t0 = time.perf_counter()
+        try:
+            text = parse_func(fpath)
+            total_chars += len(text)
+            total_files += 1
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)